app/auth.py

```diff
@@ -0,0 +1,146 @@
+from __future__ import annotations
+
+import json
+import logging
+
+from http import HTTPStatus
+from fastapi import APIRouter, Depends, Form
+from fastapi.responses import RedirectResponse
+from typing import Annotated
+
+from auth_fastapi import (
+    make_email_password,
+    email_password as core_email_password,
```

Comment:
This `core_email_password` import is mostly used to get to the various `*Response` classes defined in the core library. Should I just re-export them from the `auth_fastapi` library and import them all explicitly here?